### ISSUE TITLE
LogError at higher PU: TooManyClusters:   PhotonConversionTrajectorySeedProducerFromSingleLeg:photonConvTrajSeedFromSingleLeg

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#19803]
```
%MSG-e TooManyClusters:   PhotonConversionTrajectorySeedProducerFromSingleLeg:photonConvTrajSeedFromSingleLeg  18-Jul-2017 18:52:47 CEST Run: 299042 Event: 17281941
Found too many clusters (69349), bailing out.

```
This is visible in almost every event, apparently for somewhat large PU (>30),
e.g. in JetHT in run 299042 LS 41-42 which has PU of around 40.

@makortel @VinInn @rovere @rafaellopesdesa @Sam-Harper @jainshilpi

Please check and perhaps adjust the limits.

**Comment by slava77** [comment]
assign reconstruction

**Comment by slava77** [comment]
@makortel @VinInn @rovere @rafaellopesdesa @Sam-Harper @jainshilpi
please let me know if some activity is ongoing to address this issue.

Thank you.

**Comment by Sam-Harper** [comment]
@slava77  https://indico.cern.ch/event/616443/contributions/2669482/attachments/1496849/2329371/egammaCurrentIssues.pdf

Its noted as an E/gamma issue to be fixed (and thank you for bringing it to our attention) however we do not have active effort on this yet.

**Comment by slava77** [comment]
On 7/25/17 8:56 AM, Sam-Harper wrote:
> @slava77 <https://github.com/slava77>
> https://indico.cern.ch/event/616443/contributions/2669482/attachments/1496849/2329371/egammaCurrentIssues.pdf
> 
> Its noted as an E/gamma issue to be fixed (and thank you for bringing it
> to our attention) however we do not have active effort on this yet.

Thank you for taking note of this.
I'll keep pinging occasionally.

**Comment by slava77** [comment]
this is a weekly check on the progress to resolve this.
The effect related to the message is that the single leg seeding is turned off in these events (what's seeded by photonConvTrajSeedFromSingleLeg ).
If the module or seeding path is not essential maybe it can be removed altogether?

**Comment by Sam-Harper** [comment]
So I had a look into this. So it does rather appear that if this error is triggered, we dont get any single leg conversions. I'm debating what to do about it
1) disable this check (no other similar modules have it)
2) increase the limits by some value, this is simply now we have an extra pixel layer and so needs to be updated

Looking into the code, the layers are not BPIX4 , FPIX3 aware either which is an oversight. Will consult with the greater E/gamma community about the point of these conversion seeds.

**Comment by slava77** [comment]
+1

fixed in #20153 in 94X
